account_invoice_ubl/tests/test_ubl_generate.py

```diff
@@ -3,15 +3,79 @@
 # Copyright 2019 Onestein (<https://www.onestein.eu>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo.addons.account_tax_unece.tests.test_account_invoice import TestAccountInvoice
+from odoo.tests.common import HttpCase
 
 from ..hooks import (
     remove_ubl_xml_format_in_pdf_invoice,
     set_xml_format_in_pdf_invoice_to_ubl,
 )
 
 
-class TestUblInvoice(TestAccountInvoice):
+class TestUblInvoice(HttpCase):
```

Comment:
there's no reason to use HttpCase. Please use SavepointCase like this:

Why HttpCase? There's no controller to test here. Please revert this change and use SavepointCase and setup the class like this to reduce overhead:

```py
@classmethod
def setUpClass(cls):
    super().setUpClass()
    cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```